src/pages/staticPages/tools/mutationMapper/MutationMapperToolStore.ts

```diff
@@ -248,7 +248,7 @@ export default class MutationMapperToolStore
     }
 
     @cached get downloadDataFetcher() {
-        return new MutationTableDownloadDataFetcher(this.mutations, () => this.genomeNexusCache);
+        return new MutationTableDownloadDataFetcher(this.mutations, {}, () => this.genomeNexusCache);
```

Comment:
Is it possible to set the second parameter to `undefined`, instead of an empty object? Empty object as a parameter is not very intuitive.